### PR TITLE
Try

### PR DESCRIPTION
**Pull request by Alaa-Khattab**

### CODE REVIEW

File: database/config.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const config = {
   },
   test: {
     user: 'postgres',
-    password: '482106',
+    password: '123654',
```
Comment (saralk) [inline database/config.js]: this seems to change all the time, can we figure out a way to all use the same password

File: public/src/actions/tripsActions.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+import * as types from './actionTypes';
+import store from '../store/store';
+const getHomeTrips = (data)=>{
+  fetch('/trips',{
+    method: 'POST',
```
Comment (saralk) [inline public/src/actions/tripsActions.js]: if you're **get**ting the trips, you should use `GET` not `POST`

File: database/config.js (hunk 1 above)
Comment (Alaa-Khattab) [inline database/config.js]: fixed

File: public/src/actions/tripsActions.js (hunk 2 above)
Comment (Alaa-Khattab) [inline public/src/actions/tripsActions.js]: fixed

File: database/config.js (hunk 1 above)
Comment (Alaa-Khattab) [inline database/config.js]: we choose one password for all members

**Pull request merged**